modules/oc-mirror-troubleshooting-v2.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-troubleshooting-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 errors
+
+oc-mirror plugin v2 now logs all image mirroring errors in a separate file, making it easier to track and diagnose failures.
+
+When an image fails to mirror, and that image is mirrored as part of one or more operator bundle, oc-mirror plugin v2 notifies the user which Operators are incomplete, providing clarity on the Operator bundles affected by the error. 
+
+[IMPORTANT]
+====
+When errors occur with release or release component images, they are critical. This stops the mirroring process immediately.
+
+Errors with Operator-related images or additional images do not stop the process. Mirroring continues, and oc-mirror plugin v2 logs updates every 8 images.
+====
+
+.Procedure
+
+. Resolving server related issues:
+.. Open the `working-dir/logs` file located in the oc-mirror output directory.
+.. Look for error messages like the one above, which typically indicate server-side issues (e.g., HTTP 500 errors, expired tokens, or timeouts).
+.. Retry the mirroring process or contact support if the issue persists.
+
+. Incomplete mirroring of Operators:
+.. Check for warnings in the console or log file indicating which Operators are incomplete.
+.. If an Operator is flagged as incomplete, the image related to that Operator likely failed to mirror.
+.. You might need to manually mirror the missing image or retry the mirroring process.
+
+. Errors related to generated cluster resources:   
+.. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources (IDMS.yaml and ITMS.yaml) for the successfully mirrored images.
```

Comment:
this first list item isn't really its own step, for now I would make it a second sentence in the larger step:
```suggestion
. Errors related to generated cluster resources. Even if some images fail to mirror, oc-mirror v2 will still generate cluster resources (IDMS.yaml and ITMS.yaml) for the successfully mirrored images.
```